components/application-gateway/internal/proxy/unauthorizedresponseretrier_test.go

```diff
@@ -20,7 +20,7 @@ func TestForbiddenResponseRetrier_CheckResponse(t *testing.T) {
 		updateCacheEntryFunction := func(id string) (*CacheEntry, apperrors.AppError) {
 			return nil, nil
 		}
-		rr := newUnauthorizedResponseRetrier("", &http.Request{}, 10, updateCacheEntryFunction)
+		rr := newUnauthorizedResponseRetrier("", &http.Request{}, http.Response{}.Body, 10, updateCacheEntryFunction)
```

Comment:
`http.Response{}.Body` it does not look good and it might be quite misleading. In this case, the request body will be `nil`.

Reply:
Done!